src/otx/cli/cli.py

```diff
@@ -23,6 +23,7 @@
 from otx.cli.utils.workspace import Workspace
 from otx.core.types.task import OTXTaskType
 from otx.core.utils.imports import get_otx_root_path
+from otx.utils.utils import get_model_cls_from_config
```

Comment:
This import internally imports an `OTXModel`, which requires `torch`. The OTX CLI currently has a case where it works without torch, so the import location doesn't seem appropriate.

Reply:
done https://github.com/openvinotoolkit/training_extensions/pull/3788/commits/0adb7ea54bb57b761f75b423167b45772d08d416